draft-ietf-acme-acme.md

```diff
@@ -937,11 +938,13 @@ not recognize.  In particular, it MUST NOT reflect unrecognized fields in the
 resulting account object.  This allows clients to detect when servers do not
 support an extension field.
 
-The server SHOULD validate that the contact URLs in the "contact" field are
-valid and supported by the server.  If the client provides the server with an
-invalid or unsupported contact URL, then the server MUST return an error of type
-"invalidContact", with a description describing the error and what types of
-contact URL the server considers acceptable.
+The server SHOULD validate that the contact URIs in the "contact" field are
+valid and supported by the server. If the client provides the server with
+a contact URI using an unsupported protocol it MUST return an error of type
+"unsupportedContact", with a description describing the error and what types of
+contact URIs the server considers acceptable. The server MUST support the mailto
+scheme for contact URIs. If the client provides the server with an invalid
+contact URI, then the server MUST return an error of type "invalidContact".
```

Comment:
I reworded this in https://github.com/ietf-wg-acme/acme/pull/291/commits/44762759546b528843899bedc638426a20556677 and I think it makes it clearer that the errors are for servers that choose to validate.